test/test/spawn-bail.tap

```diff
@@ -15,8 +15,7 @@ TAP version 13
               ---
               {"found":1,"wanted":"1","compare":"===","at":{"file":"test/test/spawn.js","line":22,"column":10,"function":"foo"},"source":"tt.equal(1, '1', 'nested failure')\n"}
               ...
-            Bail out! # nested failure
-        Bail out! # nested failure
+
```

Comment:
It can be cleaned up with this change.. but it seems a little mysterious to me what the _other_ effects might be that aren't covered by tests.
```
diff --git a/lib/test.js b/lib/test.js
index e74818e..3e99084 100644
--- a/lib/test.js
+++ b/lib/test.js
@@ -578,7 +578,7 @@ function bailOnFail (self, stream, parser) {

   parser.on('assert', function (res) {
     if (!res.todo && !res.skip && !res.ok)
-      stream.emit('data', '\nBail out! # ' + res.name + '\n')
+      stream.emit('data', 'Bail out! # ' + res.name + '\n')
   })
 }

diff --git a/test/test/spawn-bail.tap b/test/test/spawn-bail.tap
index 36e92ee..ae69adf 100644
--- a/test/test/spawn-bail.tap
+++ b/test/test/spawn-bail.tap
@@ -15,7 +15,6 @@ TAP version 13
               ---
               {"found":1,"wanted":"1","compare":"===","at":{"file":"test/test/spawn.js","line":22,"column":10,"function":"foo"},"source":"tt.equal(1, '1', 'nested failure')\n"}
               ...
-
-    Bail out! # nested failure
+            Bail out! # nested failure
 Bail out! # nested failure
```